src/main/scala/org/broadinstitute/dsde/firecloud/model/DUOS.scala

```diff
@@ -1,9 +1,97 @@
 package org.broadinstitute.dsde.firecloud.model
 
-import spray.json.JsObject
+import spray.json.{JsObject, JsValue, _}
+import DefaultJsonProtocol._
 
 object DUOS {
 
+  case class DuosDataUse(
+    generalUse: Option[Boolean] = None,
+    hmbResearch: Option[Boolean] = None,
+    diseaseRestrictions: Option[Seq[String]] = None,
+    populationOriginsAncestry: Option[Boolean] = None,
+    populationStructure: Option[Boolean] = None,
+    commercialUse: Option[Boolean] = None,
+    methodsResearch: Option[Boolean] = None,
+    aggregateResearch: Option[String] = None,
+    controlSetOption: Option[String] = None,
+    gender: Option[String] = None,
+    pediatric: Option[Boolean] = None,
+    populationRestrictions: Option[Seq[String]] = None,
+    dateRestriction: Option[String] = None,
+    recontactingDataSubjects: Option[Boolean] = None,
+    recontactMay: Option[String] = None,
+    recontactMust: Option[String] = None,
+    genomicPhenotypicData: Option[String] = None,
+    otherRestrictions: Option[Boolean] = None,
+    cloudStorage: Option[String] = None,
+    ethicsApprovalRequired: Option[Boolean] = None,
+    geographicalRestrictions: Option[String] = None,
+    other: Option[String] = None,
+    illegalBehavior: Option[Boolean] = None,
+    addiction: Option[Boolean] = None,
+    sexualDiseases: Option[Boolean] = None,
+    stigmatizeDiseases: Option[Boolean] = None,
+    vulnerablePopulations: Option[Boolean] = None,
+    psychologicalTraits: Option[Boolean] = None,
+    nonBiomedical: Option[Boolean] = None
+  )
+
+  object DuosDataUse {
+    def apply(jsValues: Map[String, JsValue]): DuosDataUse = {
+      def getBoolean(f: String): Option[Boolean] = {
+        jsValues.get(f) match {
+          case x if x.isDefined => Some(jsValues(f).convertTo[Boolean])
```

Comment:
I need some better handling around the `convertTo`-s. If someone sends `"generalUse":"FOO"` instead of `"generalUse":true`, this will fail.